README.md

```diff
@@ -18,12 +18,16 @@ Visual representations of release pipelines.
 
 ## Setup
 
-    # via hokusai
+With docker:
+
     hokusai dev run 'bundle exec rake db:migrate'
     hokusai dev start
-    # or
-    dotenv bundle exec rails db:prepare
-    dotenv bundle exec rails server
+
+Or on localhost:
+
+    bundle exec rails db:prepare
+    yarn install
+    bundle exec rails server
```

Comment:
Looks like local development doesn't need any special/non-default env variables?

Reply:
Correct that's the intention.